keda/README.md

```diff
@@ -152,3 +152,35 @@ be provided while installing the chart. For example,
 ```console
 helm install keda kedacore/keda --namespace keda -f values.yaml
 ```
+
+## securityContext default configuration
+
+KEDA's default configuration tries to be safest as possible, that's why the configuration is safe as default using these values:
```

Comment:
What about this?
```suggestion
Our default configuration strives to be as secure as possible. Because of that, KEDA will run as non-root and be secure-by-default:
```